app/components/ImageBox.tsx

```diff
@@ -17,28 +33,52 @@ function ImageBox({
   className,
   tagName,
   plausibleEventName,
+  border,
+  size,
+  zoomable = true,
+  background = false,
 }: Readonly<ImageBoxProps>) {
   if (!image || !image.url) return null;
   const headingTag = tagName ? tagName : "h3";
   const imageForZoomable: ImageProps = {
     ...image,
     className: twJoin(image.className, "border-[2px] border-[#004B760D]"),
   };
+
+  const isBigImage = size === "MEDIUM" || size === "LARGE";
+
   return (
-    <figure className={twJoin("pb-[40px]", className)}>
+    <figure
+      className={twMerge(className, size && twJoin("w-full", imageSizes[size]))}
+    >
       {title && (
         <Heading className="ds-heading-03-reg pb-[24px]" tagName={headingTag}>
           {title}
         </Heading>
       )}
-      <ImageZoomable
-        image={imageForZoomable}
-        plausibleEventName={plausibleEventName}
-      />
-      <figcaption className="label-03-reg pt-[24px]">
-        {image.caption}
-      </figcaption>
-      <hr className="my-[16px] w-[120px] border-t-[2px] border-blue-300" />
+      {zoomable && isBigImage ? (
```

Comment:
Why can the image only be zoomable if it is of size MEDIUM or LARGE? Can't an image also be small (like a thumbnail) and we still want it to be zoomable? Maybe even because it is presented small by default.

Reply:
Ok. I still do not really see the point to restrict it via code. It's more code and could lead to odd behavior if the design system changes some day. But I am fine with keeping it.